explore/src/main/scala/explore/config/ModesTableCommon.scala

```diff
@@ -248,7 +257,7 @@ trait ModesTableCommon:
             val baseText = s"${r.snAt.map(_.single.value).foldMap(_.format)} / exposure"
             tooltipContent(baseText, r.ccdWarnings)
 
-        (if (r.ccdWarnings.nonEmpty)
+        (if (filterWarnings(r.ccdWarnings).nonEmpty)
```

Comment:
You can call `filterWarnings` once after line 241 and pass the result to both invocations of `tooltipContent` and this check in line 260.